ninja-db-classic/src/main/java/ninja/migrations/MigrationClassicModule.java

```diff
@@ -24,6 +26,7 @@
     @Override
     protected void configure() {
         bind(MigrationEngine.class).toProvider(MigrationEngineProvider.class);
+        //bind(FluentConfiguration.class).toProvider(FluentConfigurationProvider.class);
```

Comment:
Probably just delete this line

Reply:
Done  ( I am used  to bitbucket rather than github, so when I check in on my branch does this fix this pull request?)

Reply:
Hello @llenrup, do you think you have time to finish the last corrections requested?

Reply:
Yes sorry go ahead,
Things been busy this year

Reply:
PR is open, is it possible to close this one?

@jjlauer @llenrup